src/STW.ProcessingApi.Function.UnitTests/TestExtensions/MoqExtensions.cs

```diff
@@ -0,0 +1,42 @@
+namespace STW.ProcessingApi.Function.UnitTests.TestExtensions;
+
+using System.Net;
+using System.Text.Json;
+using Moq;
+using Moq.Language.Flow;
+using Moq.Protected;
+
+public static class MoqExtensions
+{
+    public static ISetup<HttpMessageHandler, Task<HttpResponseMessage>> SetupSendAsync(
+        this Mock<HttpMessageHandler> handler,
+        HttpMethod requestMethod,
+        string requestUrl)
+    {
+        return handler.Protected().Setup<Task<HttpResponseMessage>>(
+            "SendAsync",
```

Comment:
There are Moq extensions already that I added that let you do this. Might be best removing these in favour of those.